[Bits Dev] Add troubleshooting case for workflows:write issue with GH (#32964)
DataDog/documentation: content/en/bits_ai/bits_ai_dev_agent/setup.md

**Note**: For best results, add a [custom instructions file](#configure-custom-instructions) (like `claude.md`) to your repository with instructions on how to build and test your code.

## Troubleshooting

### Creation of PRs fails unexpectedly

> In some cases, especially in repositories with many branches, GitHub fails to run the permission check when creating the branch for the session. If you're using a custom GitHub app, a known workaround is to add the `workflows:write` permission to your app in [Source Code Integration][5]. Beware that this will allow Bits AI to create workflows on your repository, which has security implications.

Reviewer: ```suggestion
In some cases, especially in repositories with many branches, GitHub fails to run the permission check when creating the branch for the session. If you're using a custom GitHub app, you can work around this by adding the `workflows:write` permission to your app in [Source Code Integration][5].

**Note**: This allows Bits AI to create workflows on your repository, which has security implications.
```

**Note**: For best results, add a [custom instructions file](#configure-custom-instructions) (like `claude.md`) to your repository with instructions on how to build and test your code.

## Troubleshooting

### Creation of PRs fails unexpectedly

In some cases, especially in repositories with many branches, GitHub fails to run the permission check when creating the branch for the session. If you're using a custom GitHub app, you can work around this by adding the `workflows:write` permission to your app in [Source Code Integration][5].

> **Note**: This allows Bits AI to create workflows on your repository, which has security implications.

Reviewer: ```suggestion
**Note**: This permission allows Bits AI to create workflows in your repository and has security implications.
```
